tests/test_to_process.py

```diff
@@ -73,6 +73,7 @@ async def test_cancel_before() -> None:
     pytest.raises(LookupError, to_process._process_pool_workers.get)
 
 
+@pytest.mark.usefixtures("deactivate_blockbuster")
 async def test_cancel_during() -> None:
```

Comment:
There's an error when calling 
```sh
  | exceptiongroup.ExceptionGroup: unhandled errors in a TaskGroup (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/home/runner/work/anyio/anyio/src/anyio/to_process.py", line 95, in run_sync
    |     request = pickle.dumps(("run", func, args), protocol=pickle.HIGHEST_PROTOCOL)
    | AttributeError: Can't pickle local object '_wrap_blocking.<locals>.wrapper'
    +------------------------------------
```

`_wrap_blocking.<locals>.wrapper` is a blockbuster function (the one that wraps the framework core functions).
Not sure what to do so just deactivating blockbuster for this test.